djmoney/models/fields.py

```diff
@@ -34,12 +34,17 @@
 if 'djmoney_rates' in settings.INSTALLED_APPS:
     try:
         from djmoney_rates.utils import convert_money
-        AUTO_CONVERT_MONEY = True
+        RATES_INSTALLED = True
     except ImportError:
         # NOTE. djmoney_rates doesn't support Django 1.9+
-        AUTO_CONVERT_MONEY = False
+        RATES_INSTALLED = False
 else:
-    AUTO_CONVERT_MONEY = False
+    RATES_INSTALLED = False
+
+
+if getattr(settings, 'DJMONEY_AUTO_CONVERT_MONEY', False):
```

Comment:
This line is duplicated (at line 79), so another constant with ``getattr(settings, 'DJMONEY_AUTO_CONVERT_MONEY', False)`` will be useful